.github/CODEOWNERS

```diff
@@ -0,0 +1,58 @@
+# Lines starting with '#' are comments.
+# Each line is a case-sensitive file pattern followed by one or more owners.
+# Order is important. The last matching pattern has the most precedence.
+# More information: https://docs.github.com/en/articles/about-code-owners
+#
+# Please mirror the repository's file hierarchy in case-sensitive lexicographic
+# order.
+
+# .github
+/.github/CODEOWNERS @shahmishal @swiftlang/core-team 
+
+
+### Evolution process documentation
+
+# The primary Swift Evolution process documentation file.
+/process.md @rjmccall
+
+
+### Visions
+
+# The top-level "visions" directory.
+#
+# Approval from the Core Team is required to add a new vision document.
+/visions/ @swiftlang/core-team
```

Comment:
This is not actually true; visions are products of the evolution workgroups just like proposals are. It's just not as high-traffic as `proposals/` is.